autoload/fuzzy/ag.vim

```diff
@@ -124,9 +124,14 @@ def AgJobStart(pattern: string)
         out_cb: function('JobHandler'),
         out_mode: 'raw',
         exit_cb: function('ExitCb'),
+        err_cb: function('ErrCb'),
     })
 enddef
 
+def ErrCb(channel: channel, msg: string)
+    echoerr msg
```

Comment:
This causes problems with `ag` and `--max-count` as it emits an error, fixed in later commit by adding `--silence` to `ag` options (doesn't silence critical errors like invalid options)